### ISSUE TITLE
Mod Suggestions

### ISSUE THREAD

**Issue by whatthedrunk** [whatthedrunk/allthemods#173]
Cleaning up the last post on this. Here is everything suggested and the progress with it.

- [x] IC2
- [x] ftb lib and ftb utilities 
- [ ] Daedalus Labyrinth
- [ ] Chicken Chunks
- [ ] Silimatics
- [ ] Quantum Flux
- [ ] Cyberware (It looks like it might come back)
- [ ] Login Shield (This can be added again in the near future)

**List of mods that wont be added at this time.**
Veinminer

**Comment by RocketPhoenix** [comment]
So new suggestions go here?
If possible, there are a few cool mods that I think would really fit the pack:
-[Solar Flux Reborn](https://minecraft.curseforge.com/projects/solar-flux-reborn) (as an RF counterpart to compact solars, and possibly an energy converter in a future update)
-[Totemic](https://minecraft.curseforge.com/projects/totemic) (It's fun)
-[Woot](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2659457-woot-the-alternative-mob-factory-1-9-4-0-3-0-beta) (I haven't played with it yet, but it's an interesting idea that I would love to try in a mod pack environment)
-[Inventory Pets](https://mods.curse.com/mc-mods/minecraft/229380-inventory-pets) (They're ADORABLE)
-[Realistic Terrain Gen](https://minecraft.curseforge.com/projects/realistic-terrain-generation) (I've loved this mod since it's 0.0.17 release and it is absolutely beautiful)
-[Streams](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2346379-streams-real-flowing-rivers) (This is just a shot in the dark, but if what WTFcore did to world generation was acceptable, than this might be as well. Plus it looks lovely)
-[Terraqueous](https://minecraft.curseforge.com/projects/terraqueous) (It's neat)
-[Minecraft Flight Simulator](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2597098-minecraft-flight-simulator-the-most-realistic) (An amazing counterpart to ViesCraft. I had a ton of fun flying, as well as a good laugh at how janky it moved. ps: don't activate a piston into a plane, it will cause problems)
-[OpenFM](https://minecraft.curseforge.com/projects/openfm) (If only one mod on this list gets chosen, let it be this one. Open Computers is already in this pack, and this mod only adds 3 items. pls)
There were a few others that caught my eye, but I haven't played them, and this post needs to end.
Thank you for your consideration!
d(- - )

**Comment by desht** [comment]
Would you also consider adding TOP Addons (https://minecraft.curseforge.com/projects/top-addons?gameCategorySlug=mc-mods&projectID=247111) ?

For those of us who prefer TOP to Waila, especially in conjunction with Storage Drawers :)

**Comment by capSAR273** [comment]
I wrote a suggestion on the subreddit, Dan said to post here also: https://www.reddit.com/r/allthemods/comments/523zwb/suggestion_to_set_up_irc_for_ssp/

Its to add an IRC mod to connect so singleplayer people can talk to one another, like the TPPI and TPPI2 modpacks have. I'm messing with an IRC mod right now to see if its easy to set up.

**Comment by chaos234** [comment]
Hey, I am missing Backup Mods which are highly recommanded. Also a nice mining age will be fine.

So I suggest Aroma1997 onse (Core, Backup, Dimensional World). Here are the Downloads for it:

https://mods.curse.com/mc-mods/minecraft/223735-aroma1997core#t1:other-downloads
https://mods.curse.com/mc-mods/minecraft/aroma1997s-dimensional-world#t1:other-downloads
https://mods.curse.com/mc-mods/minecraft/225658-aromabackup#t1:other-downloads

And I also think that you should supply Uncomplicated from him, link: http://forum.industrial-craft.net/index.php?page=Thread&threadID=11586

With this, u can use the old E-Net instead of the new ones. There are also some other nice functions but at this point it is only to have the old IC2 Network with all the nice explosions back :).

Hmmm, to bad that Traveler's Gear is only 1.7.10 and Thaumcraft actually only 1.8.9 :( ... so thi is all what I have for you.

**Comment by DFliyerz** [comment]
https://minecraft.curseforge.com/projects/tinkers-tool-leveling

**Comment by Hirsu** [comment]
Hi,

McJty from RFTools has created a new mod, RFTools Control.
It's an automation mod which functions somewhat like Steve's Factory Manager used to.
You create your own programs by connecting tiles with certain functions (a bit like Psi).

The mod is still in beta phase, but McJty is actively swatting bugs left and right.
His mods are always well-developed and maintained; I think this latest addition should be up for consideration for All The Mods!

For a link to the Curse page, [click here](https://minecraft.curseforge.com/projects/rftools-control).

Thanks!

**Comment by Ipsis** [comment]
Since I'm the mod author of Woot, I'm a bit concerned you were getting crashes, since it is running in other packs without issue. You don't happen to have any crash information do you or the version that you were trying? Basically, if there are issues with the mod I'd like to get them sorted.

**Comment by ExpensiveKoala** [comment]
[Flux Networks](https://minecraft.curseforge.com/projects/flux-networks) - Kinda like tesseracts but better. Also made by the same guy who makes the Calculator mod.

**Comment by krasnyineko** [comment]
I second [RFTools Control](https://minecraft.curseforge.com/projects/rftools-control) as it seems to be [out of beta now](https://www.reddit.com/r/feedthebeast/comments/53s7t0/rftools_control_is_out_of_beta_100_release/).

**Comment by krasnyineko** [comment]
I would like to throw in there [Flat Coloured blocks](https://minecraft.curseforge.com/projects/flat-colored-blocks) for the use in chisel and just building in general.

**Comment by calvo2** [comment]
Hello, Just wanted to add a couple of mod suggestions.

Morph O Tool and Akashic Tome.  Both mods are by Vazkii and each provides a way to combine tools and mod books respectively.
[https://minecraft.curseforge.com/projects/morph-o-tool?gameCategorySlug=mc-mods&projectID=245287](url)

[https://minecraft.curseforge.com/projects/akashic-tome?gameCategorySlug=mc-mods&projectID=250577](url)
Soon to be released Super Circuit Maker would be awesome to have also.

Sad to hear about WOOT issues.  Would love to see that mod added also.

**Comment by DFliyerz** [comment]
https://minecraft.curseforge.com/projects/magneticraft

**Comment by calvo2** [comment]
Hello again,  Would it be possible to have Ender Utilities added.

[https://mods.curse.com/mc-mods/minecraft/224320-ender-utilities](url)

**Comment by DnTheMn** [comment]
(Ohound dont know where to put things) 
"It seems that Ollie removed all the flux network functionality from Calculator to move it to a new mod

https://minecraft.curseforge.com/projects/flux-networks"

**Comment by brodycas3** [comment]
Could you possibly look into adding these mods?

Mo Creatures - https://minecraft.curseforge.com/projects/mo-creatures
Better Achievements - Must have - https://minecraft.curseforge.com/projects/betterachievements
Mo' Villages - https://minecraft.curseforge.com/projects/mo-villages
Ding - https://minecraft.curseforge.com/projects/ding

Please look over these mods and think about adding them. Especially Better Achievements that's a must have. BTW I love this pack keep up the good work.

**Comment by Towsty** [comment]
MattDahEpic Core

Adds some very handy commands, most notably a pregen. Would love to see this added to ATM and ATM:Expert

**Comment by SephirothWS** [comment]
Curseforge link - https://minecraft.curseforge.com/projects/mob-grinding-utils

I am requesting this mod be added to the pack to facilitate with more creative mob grinding designs, for one, but the most important block(s) are the Wither & Ender Dragon mufflers for multiplayer. As I've been playing on a multiplayer server, I got an automated Wither killer set up, and people moan and groan about the constant thump noise from the withers because Vanilla stupidly made that sound global.

**Comment by OrdinatorStouff** [comment]
Realistic Terrain Gen and BoP work together. It takes a few config file changes to but otherwise RTG can honor BoP's biomes works well. You can even have BiomeTweaker in too (all three). Or was there some other reason not to add it that I'm not thinking of?

**Comment by Fortanono** [comment]
Can you add Nature's Compass to ATM1 as well as Expert? It would really be nice.

**Comment by Poshtester63** [comment]
Since thermal expansion has just released, would you be adding that to the pack as well.
Also would be nice to see advanced rocketry in the pack.

**Comment by ozhound** [comment]
TE has been added to ATM1.

**Comment by Poshtester63** [comment]
Yeah I saw that. Any thoughts on Advanced Rocketry? And I assume Thaumcraft 6 will be added when its released as well?

**Comment by ozhound** [comment]
Thaumcraft definitly, advanced rocketry, probably not. I think it's been
asked before.

**Comment by Poshtester63** [comment]
Okay, another mod suggestion list!

[ET Lunar](https://minecraft.curseforge.com/projects/et-lunar-environmental-tech-addon) - Why not?
[WIRED: More Augs](https://minecraft.curseforge.com/projects/wired-more-augs)
[Matter Overdrive](https://mods.curse.com/mc-mods/minecraft/229694-matter-overdrive) - It's in Beta though

Also, why have EnderTanks when Ender Storage has all of its functionality?
Same thing with all the backpack/bags mods?
I understand that this _is_ a kitchen pack, but its just pointless redundancy to have multiple mods which have roughly **almost the same** functionality. (Mods which overlap, but are distinct from each other, such as most of the tech mods, aren't counted as redundancy in this statement)
